slobrok/src/vespa/slobrok/server/local_rpc_monitor_map.cpp

```diff
@@ -93,6 +93,7 @@ void LocalRpcMonitorMap::addLocal(const ServiceMapping &mapping,
         if (exists.spec == mapping.spec) {
             LOG(debug, "added mapping %s->%s was already present",
                 mapping.name.c_str(), mapping.spec.c_str());
+            // clear exists.ignoreFirstOk ?
```

Comment:
another question; should we really return 'ok' when we known the service is in the process of being removed?